src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -404,19 +404,24 @@
     <Field Name="Right" Type="BoundExpression"/>
 
     <Field Name="DeconstructSteps" Type="ImmutableArray&lt;BoundDeconstructionDeconstructStep&gt;" Null="NotApplicable" SkipInVisitor="true" />
```

Comment:
For now this is fine.

---
In reply to: [75046767](https://github.com/dotnet/roslyn/pull/13115#discussion_r75046767) [](ancestors = 75046767)